packages/service/tests/unit/test_discovery_client.py

```diff
@@ -393,3 +394,7 @@ def _assert_service_info_equal(
     assert set(expected.provided_interfaces) == set(actual.provided_interfaces)
     assert expected.service_class == actual.service_class
     assert expected.annotations == actual.annotations
+    if isinstance(actual, GrpcServiceDescriptor):
```

Comment:
Why the if since the asserts are the same?

Reply:
sloppiness - they didn't used to be the same, and I forgot to update the check.  Updated now.